SofaKernel/modules/SofaMeshCollision/src/SofaMeshCollision/BarycentricPenalityContact.h

```diff
@@ -136,4 +136,4 @@ extern template class SOFA_SOFAMESHCOLLISION_API BarycentricPenalityContact<Cyli
 extern template class SOFA_SOFAMESHCOLLISION_API BarycentricPenalityContact<CylinderCollisionModel<sofa::defaulttype::Rigid3Types>, OBBCollisionModel<sofa::defaulttype::Rigid3Types>>;
 #endif
 
-} // namespace sofa::component::collision
+} //namespace sofa::component::collision
```

Comment:
space removed not needed

Reply:
this comes from a script and this is not the only one affected... a bit lazy to correct all 🤐